packages/ember-console/lib/index.js

```diff
@@ -56,7 +22,7 @@ export default {
    @param {*} arguments
    @public
   */
-  log: consoleMethod('log') || K,
+  log: console.log, // eslint-disable-line no-console, no-undef
```

Comment:
I'm _slightly_ torn here, I don't know if we can use `console.log` unbound from `console` for all supported platforms.  Do you know?

The safe bet here is essentially to do:

```js
log() { return console.log(...arguments); }
```

Reply:
Tried `var log = console.log; log('foo');` in Safari, Firefox, Chrome and IE 11. Worked everywhere.  
I can change it to be safe though.

Reply:
Checkout this MS Edge screenshot: 

![MS Edge failure](https://assets-proxy.saucelabs.com/jobs/fae1d7cded1d47cf8ab72a2e179b6cfc/0275screenshot.png)

I think this is for `console.debug`?

Reply:
@t-sauer - No rush here, just curious if you had time to review?

Reply:
I am on it right now. Seems like `.bind(console)` is indeed needed in some cases.